### DISCUSSION TITLE
How to transform db column to different json field

### DISCUSSION THREAD

**Discussion by letsautom8** [fastapi/fastapi#7377] [Questions]
Im new to Fast API. I have followed the guide on sql databases and got it to work. Thank you for this awesome framework. 
Just a question on how do I present the json fields using different field name. This is because there is a requirement to hide the actual column name of the database.
db column: 
building_id

expected json response
buildingId

**Comment by letsautom8** [comment]
Sorry. Im closing. This is is sqlalchemy question. For those who want to map their db field to a different name use:
```
class User(Base):
    __tablename__ = 'user'
    id = Column('user_id', Integer, primary_key=True)
    name = Column('user_name', String(50))
```

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue :+1:
